cocos/editor-support/spine/SkeletonRenderer.cpp

```diff
@@ -61,7 +61,7 @@ SkeletonRenderer* SkeletonRenderer::createWithFile (const std::string& skeletonD
 }
 
 void SkeletonRenderer::initialize () {
-	_worldVertices = new float[1000]; // Max number of vertices per mesh.
```

Comment:
The constant is predefined in some Spine's frameworks. You can't just change it, because you don’t have enough vertices in the model, just because the artist made such a reach and expensive skeleton, it will affect the global changes of the whole Spine (for example: in SkeletonRenderer.m had another reference for it: _worldVertices = MALLOC(float, 1000); // Max number of vertices per mesh. and SkeletonExtensions.cs hava similar).

Reply:
Only one solution - you've to submit PR to spine repo . As  talking several times before, and [last time here](https://github.com/cocos2d/cocos2d-x/pull/19163#issuecomment-451872989). Any of PR which referenced to Spine runtime would be closed due to Spine license:

> The cocos2d-x can't modify spine runtime due to it's license limit. you should post PR to spine runtime repo: https://github.com/EsotericSoftware/spine-runtimes